WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -64,6 +64,8 @@ protected override async Task ActionAsync(CancellationToken cancel)
 
 		private void StepInputRegistrationPhase()
 		{
+			Logger.LogInfo("Starting Input Registration Phase.");
```

Comment:
What about:

```
Logger.LogInfo("Starting InputRegistration Phase.");
```
?

For me this is actually more readable, not sure how others see it.